fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -776,21 +776,10 @@ public static Boolean analyzeDisableAutoCompaction(Map<String, String> propertie
     }
 
     public static Boolean analyzeVariantFlattenNested(Map<String, String> properties) throws AnalysisException {
-        if (properties == null || properties.isEmpty()) {
```

Comment:
better add a config（default false）， if poc need this

Reply:
test need to enable the config(session var), see `disable_inverted_index_v1_for_variant`